causalml/inference/meta/rlearner.py

```diff
@@ -223,7 +228,33 @@ def estimate_ate(self, X, p, treatment, y):
             ate_lb[i] = _ate_lb
             ate_ub[i] = _ate_ub
 
-        return ate, ate_lb, ate_ub
+        if not return_ci:
+            return ate, ate_lb, ate_ub
+        else:
+            start = pd.datetime.today()
+            self.t_groups_global = self.t_groups
+            self._classes_global = self._classes
+            self.model_mu_global = deepcopy(self.model_mu)
+            self.models_tau_global = deepcopy(self.models_tau)
+            ate_bootstraps = np.zeros(shape=(self.t_groups.shape[0], n_bootstraps))
+
+            for n in range(n_bootstraps):
+                ate_b = self.bootstrap(X, p, treatment, y, size=bootstrap_size)
+                ate_bootstraps[:, n] = ate_b.mean()
+                if verbose and n % 10 == 0 and n > 0:
+                    now = pd.datetime.today()
+                    lapsed = (now-start).seconds
+                    logger.info('{}/{} bootstraps completed. ({}s lapsed)'.format(n, n_bootstraps, lapsed))
+
+            ate_lower = np.percentile(ate_bootstraps, (self.ate_alpha / 2) * 100, axis=1)
+            ate_upper = np.percentile(ate_bootstraps, (1 - self.ate_alpha / 2) * 100, axis=1)
+
+            # set member variables back to global (currently last bootstrapped outcome)
+            self.t_groups_global = self.t_groups
+            self._classes_global = self._classes
+            self.model_mu_global = deepcopy(self.model_mu)
+            self.models_tau_global = deepcopy(self.models_tau)
```

Comment:
These should be flipped:
```
self.t_groups = self.t_groups_global
self._classes = self._classes_global
self.model_mu = deepcopy(self.model_mu_global)
self. models_tau = deepcopy(self. models_tau_global)
```
BTW @yungmsh, is there any reason why we keep `_global` variables as member variables? If not, let's not use `self.`.